### PR TITLE
Solve library not support intl v 0.18 

### PR DESCRIPTION
**Pull request by the-best-is-best**

### CODE REVIEW

**Comment by bdlukaa** [comment]
Nice catch! Please do not update the version of the package

**Comment by the-best-is-best** [comment]
> Nice catch! Please do not update the version of the `fluent_ui` package

ok i return version of fluent_ui and remove all changed in changelog file

File: pubspec.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: fluent_ui
 description: Implements Windows UI in Flutter. Based on the official documentation
-version: 4.5.0
+version: 4.5.1
```
Comment (bdlukaa) [inline pubspec.yaml:3]:
```suggestion
version: 4.5.0
```

this should be maintained as it is

Comment (the-best-is-best) [reply pubspec.yaml]: Ok sorry i solved it

**Review by WinXaito — APPROVED** [review]
Lgtm

**Pull request merged**